common/deposits/btr_citadel_species_planetary_deposits.txt

```diff
@@ -228,6 +228,156 @@ btr_d_asrep_tidal_barrage = {
 		district_generator_max  = 2
 	}
 	
+	drop_weight = {
```

Comment:
I'll be doing a full more in depth review of planetary deposits and their balance as I go through the map expansion and get everything else Jon had planned out implemented, but this all looks good.